resources/ext/cli_defaults/cli-defaults.sh.erb

```diff
@@ -7,4 +7,4 @@ if [ ! -e "$JRUBY_JAR" ]; then
   return 1
 fi
 
-CLASSPATH="${CLASSPATH}:${JRUBY_JAR}"
+CLASSPATH="${CLASSPATH}:${JRUBY_JAR}:/opt/puppetlabs/server/data/<%= EZBake::Config[:real_name] %>/jars/*"
```

Comment:
Generally I'd expect anything under data to be, well, data and not something to be executed - but I'm not sure there's a better choice for this. We should validate the permissions that this gets created with, they should be fairly restrictive for write access, at least.